packages/jest-each/src/index.js

```diff
@@ -0,0 +1,20 @@
+import arrayEach from './array';
+import templateEach from './template';
+
+const each = (...args) => {
+  if (args.length > 1) {
+    return templateEach(global)(...args);
+  }
+
+  return arrayEach(global)(...args);
+};
+
+each.withGlobal = g => (...args) => {
```

Comment:
you could probably just export
```js
export default each.withGlobal(global);
```

Reply:
I think it's more obvious to just export the `each`, as it is now